e2e/test/netstandard20/ProvisioningE2ETests.cs

```diff
@@ -50,274 +47,664 @@ public enum EnrollmentType
             Group
         }
 
+        public enum TransportProtocol
```

Comment:
Why are you introducing this enum instead of using existing one?

Reply:
I'll see if I can rewrite these tests to reuse that enum, sure